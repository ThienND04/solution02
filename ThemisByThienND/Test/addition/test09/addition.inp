10
1807 2249 2073 984943658
1930 2272 3544 957850878
1923 2709 2440 915438165
1492 2042 1987 937522503
1327 2729 2840 943542612
1303 2169 3709 998097157
1560 1933 1099 904280278
1816 2335 1097 936807826
1512 2267 3810 904877633
1979 2149 1579 984558821
