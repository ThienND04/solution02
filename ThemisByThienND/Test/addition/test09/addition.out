57576938
59038776
32965186
72595983
43281617
47517979
123374151
94333099
31349337
72175784
